integration/src/main/java/ai/djl/integration/tests/ndarray/NDIndexTest.java

```diff
@@ -52,6 +53,22 @@ public void testPick() {
         }
     }
 
+    @Test
+    public void testGather() {
+        // Currently in windows gradle cannot find all the engines.
```

Comment:
Is this comment still valid?

Reply:
Probably. It seems like something is wrong with `runtimeOnly` dependencies on windows gradle.

Reply:
Yes after updating gradle, on GitHub the test stills fails for the same reason